master/reference/advanced/etcd-rbac/kubernetes-advanced.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Advanced Kubernetes set ups utilizing etcd RBAC
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes advanced configuration
+options of individual component that can be configured and offers some help
```

Comment:
So, my comments are a little all over the place - apologies.

I think one problem I find with this doc is that it is an extension of the simple guide but it also feels like it's trying to be a standalone guide - it still talks about the kube server requiring a cert in the  "Components that need etcd roles" but then refers back to the simple guide.  Can we just talk about the deltas in this doc - i.e. you can if you want configure each of the individual Calico components with different roles to further limit their etcd access, and in particular, limit which actions users of calicoctl can perform. blah blah.